src/index.ts

```diff
@@ -45,15 +51,31 @@ async function forceSymlink (
 ): Promise<{ reused: boolean, warn?: string }> {
   let initialErr: Error
   try {
-    await fs.symlink(target, path, symlinkType)
+    if (symlinkPermissionCheckDone) {
+      await fs.symlink(resolveSrc(target, path), path, symlinkType)
+    } else {
+      try {
+        await fs.symlink(resolveSrc(target, path), path, symlinkType)
+        symlinkPermissionCheckDone = true
+      } catch (err) {
```

Comment:
it can be simplified 
```suggestion
      try {
        await fs.symlink(resolveSrc(target, path), path, symlinkType)
      } catch (err) {
        if (symlinkPermissionCheckDone) {
```

Reply:
For the second and subsequent executions, we want to avoid entering the `try` block. Shall I add a comment to explain it?

Reply:
I will refactor code around here to avoid extra virtual calls.